examples/helpers/array_equality.js

```diff
@@ -0,0 +1,19 @@
+Array.isEqual = function (array1, array2) {
+    if (!array1 || !array2)
+        return false
+
+    if (array1.length !== array2.length)
+        return false
+
+    for (i = 0, l = array1.length; i < l; i++) {
+
+        if (array1[i] instanceof Array && array2[i] instanceof Array) {
+            if (Helper.isEqual(array1[i], array2[i]))
```

Comment:
@paulomilitante I think this should be `Array.isEqual`

Reply:
@adimasuhid 
Ah, I missed this one too. I changed it to Array.isEqual. Thanks!